.github/scripts/commit-message.js

```diff
@@ -0,0 +1,9 @@
+const fs = require('fs');
+
+const data = `${process.env.JIRA} ${process.env.DESCRIPTION}\n\n${process.env.DESCRIPTION}`;
```

Comment:
What's the benefit of repeating the description?

Reply:
It shows up in the PR description, I agree it is redundant.... I will remove this.